addons/website_slides/models/slide_channel.py

```diff
@@ -24,6 +25,7 @@ class ChannelUsersRelation(models.Model):
     channel_id = fields.Many2one('slide.channel', index=True, required=True, ondelete='cascade')
     completed = fields.Boolean('Is Completed', help='Channel validated, even if slides / lessons are added once done.')
     completion = fields.Integer('% Completed Slides')
+    completion_date = fields.Datetime('Completion date', help='Completion date')
```

Comment:
Should be done in a specific commit, to avoid having both digest and slides changes in the same commit (can be considered as separate small features).

Reply:
Moved in a dedicated commit using a compute method as suggested.